sympy/functions/elementary/exponential.py

```diff
@@ -1055,6 +1055,9 @@ def _eval_as_leading_term(self, x, logx=None, cdir=0):
             return self.func(x0) - 2*I*S.Pi
         return self.func(arg)
 
+    def is_constant(self, *wrt, **flags):
+        return self.args[0].is_constant(*wrt, **flags)
```

Comment:
Why is this needed for `log`? No other function class has this.

Reply:
It was imported when fixing test_issue_22020 in test_expr.py
Seems without it that case can still pass now. I'll remove it to see what CI says.